app/src/main/java/ru/firstSet/kotlinOne/FragmentMoviesList.kt

```diff
@@ -0,0 +1,50 @@
+package ru.firstSet.kotlinOne
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import androidx.fragment.app.Fragment
+
+class FragmentMoviesList() : Fragment() {
+    private var someFragmentClickListener: SomeFragmentClickListener? = null
+    private var fmlCombinedShape: View? = null
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? = inflater.inflate(R.layout.fragment_movies_list, container, false)
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        fmlCombinedShape = view.findViewById<View>(R.id.fmlCombinedShape).apply {
```

Comment:
@kamar19 Looks like it is a wrong view to set listener to it. We need to handle click on the list item.